fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -90,14 +90,22 @@ object ItemKicker {
     case _ => None
   }
 
+  def stringIfPlainText(string: String) : Option[String] = {
+    val StringWithHtml = "<\\w+>".r.unanchored
```

Comment:
Good catch about the regex.  
The problem with `scala.xml.XML.loadString` is that it does not determine whether something contains HTML tags, but rather whether it is well-formed XML (a much narrower specification). This is not particularly useful for this task.